src/js/view/view.js

```diff
@@ -351,6 +351,12 @@ define([
                 // area around scrollbar on the playlist.  skin fix required to remove
                 '.jw-playlist-container ::-webkit-scrollbar'
             ], 'border-color', backgroundColor);
+
+            // Set global colors, used by related plugin
+            utils.css('#' + id + ' .jw-color-active', { color: activeColor, borderColor: activeColor, stroke: activeColor }, id);
+            utils.css('#' + id + ' .jw-color-active-hover:hover', { color: activeColor, borderColor: activeColor, stroke: activeColor }, id);
+            utils.css('#' + id + ' .jw-color-inactive', { color: inactiveColor, borderColor: inactiveColor }, id);
+            utils.css('#' + id + ' .jw-color-inactive-hover:hover', { color: inactiveColor, borderColor: inactiveColor, stroke: inactiveColor }, id);
```

Comment:
Use addStyle:

These are the classes that get `activeColor` applied to an element's color.
```js
             // Apply active color
             addStyle([
                '.jw-color-active',
                '.jw-color-active-hover:hover',
                 // Toggle and menu button active colors
                 '.jw-toggle',
                 '.jw-button-color:hover'
             ], 'color', activeColor);
```
You'll need to add a block for each combo of active/inactive and stroke/border.